src/common/id/SegmentId.cpp

```diff
@@ -0,0 +1,55 @@
+/* Copyright (c) 2021 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License.
+ */
+#include "common/id/SegmentId.h"
+
+#include <cstdint>
+#include <mutex>
+
+#include "common/base/Logging.h"
+
+namespace nebula {
+
+int64_t SegmentId::getId() {
```

Comment:
I think getId() is better to return  StatusOr<int64_t> so that when something goes wrong, the error message can be returned in time instead of just LOG(ERROR).  eg. when clock back occurred in snowflake or fetchSegment() rpc failed in segment.